packages/printable-itinerary/src/access-leg.js

```diff
@@ -2,15 +2,24 @@ import coreUtils from "@opentripplanner/core-utils";
 import { humanizeDistanceString } from "@opentripplanner/humanize-distance";
 import PropTypes from "prop-types";
 import React from "react";
+import AccessibilityAnnotation from "./accessibility-annotation";
 
 import * as S from "./styled";
 
-export default function AccessLeg({ config, leg, LegIcon }) {
+export default function AccessLeg({
+  accessibilityScoreGradationMap,
+  config,
+  leg,
+  LegIcon
+}) {
   return (
     <S.Leg>
-      <S.ModeIcon>
-        <LegIcon leg={leg} />
-      </S.ModeIcon>
+      <AccessibilityAnnotation
+        accessibilityScoreGradationMap={accessibilityScoreGradationMap}
+        grayscale
```

Comment:
See my comment about using a dark border instead of the light grey background.

Reply:
Good call! Updated in 9b052113f918c4d8c638f99fb623ad71b8bba15d